### ISSUE TITLE
Client does not connect

### ISSUE THREAD

**Issue by fkooman** [Amebis/eduVPN#66]
Report from a user:

- click on profile to connect to VPN
- connect screen appears
- after 3 seconds the "main" screen reappears (without error)

**Comment by rozmansi** [comment]
I cannot reproduce this problem. Can this particular user reproduce it?

After redirected back to the "main" page, does OpenVPN connect in the background? I guess not.

Possibility 1: openvpn.exe process dies.

Possibility 2: eduVPN Client <> OpenVPN Interactive Service communication silently fails. Reviewing the code, I found that I have exceptions in the first communication phase covered and an error is reported correctly. However, the second phase still does set the error property in case of an exception but the eduVPN Client doesn't check this property. I fixed that in 615e6a011395f62a8cd6fcabe33363fe39d6ef2b now.
To summarize: if the error is repeatable, the user still won't be able to connect. However, the exception details should be displayed now, allowing us to continue resolving this issue from there.

Possibility 3:
User double clicks on the profile which is displayed at approximately the same location as the _Diconnect_ button. The first click triggers the connection. Page changes and the second click is sent to the _Disconnect_ button.

**Comment by fkooman** [comment]
I also never saw this particular behavior. So maybe it was bad luck. Good you added some more error checking. I only reported it here because maybe based on the symptoms you could find out what it could be...
